docs/content/querying/granularities.md

```diff
@@ -52,5 +309,106 @@ This will bucket by two-day chunks in the Pacific timezone.
 
 This will bucket by 3-month chunks in the Pacific timezone where the three-month quarters are defined as starting from February.
 
+#### Example
+
+Reusing the data in the previous example, if you submit a groupBy query with 1 day period in Pacific timezone,
+
+``` json
+{
+   "queryType":"groupBy",
+   "dataSource":"my_dataSource",
+   "granularity":{"type": "period", "period": "P1D", "timeZone": "America/Los_Angeles"},
+   "dimensions":[
+      "language"
+   ],
+   "aggregations":[
+      {
+         "type":"count",
+         "name":"count"
+      }
+   ],
+   "intervals":[
+      "2000-01-01T00:00Z/3000-01-01T00:00Z"
```

Comment:
can you change the interval also to America/Los_Angeles timezone? that would be the common use case.